do-catalog/src/components/ModalSubscription.vue

```diff
@@ -129,38 +248,17 @@ export default {
   cursor: pointer;
 }
 
-.data-request-card {
-  padding: 16px 16px 20px 24px;
-  border: 1px solid $neutral--300;
-  border-radius: 4px;
-  background-color: $white;
+.u-align--center {
+  text-align: center;
```

Comment:
A similar class should exists somewhere in the general styles, @albertoac do you know if it exists?

Reply:
This class was already here before this PR so I guess it shouldn't exist

Reply:
Yes, and I should be the one that put it there :sweat_smile: 
That's why I asked @albertoac

Reply:
I believe that was my fault 😓 
I think at the moment I searched and didn't found any class to just center the text, so I added it there, but maybe it's better to move it to the utilities or typography scss.

Reply:
No worries! Thanks @albertoac